data/mods/Generic_Guns/bandoliers/bandolier.json

```diff
@@ -70,6 +78,7 @@
           "shotcanister": 60,
           "paintball": 60
         },
+        "rigid": true,
```

Comment:
The base `ammo_pouch.json` and `bandolier.json` items' `pocket_data` do not carry the `rigid` flag, so these probably shouldn't either.

Reply:
Yes, you are correct! I am quite new to using PR:s on Github, so I just made a new commit with the changes, was this correct? Thanks